stable_baselines3/common/monitor.py

```diff
@@ -163,6 +158,42 @@ class LoadMonitorResultsError(Exception):
     pass
 
 
+class ResultsWriter:
+    """
+    A result writer that saves the data from the `Monitor` class
+
+    :param filename: the location to save a log file, can be None for no log
+    :param header: the header dictionary object of the saved csv
+    :param reset_keywords: the extra information to log, typically is composed of
+        `reset_keywords` and `info_keywords`
+    """
+
+    def __init__(
+        self,
+        filename: str = "",
+        header: Dict[str, Union[float, str]] = {},
```

Comment:
please avoid mutable as default, see https://github.com/satwikkansal/wtfpython#-beware-of-default-mutable-arguments

Reply:
Should I use `None` instead maybe?

Reply:
yes, the classic way of doing it is:

```python
if header is None:
    header = {}
```